sycl/test-e2e/Graph/Inputs/host_task2_multiple_roots.cpp

```diff
@@ -3,7 +3,7 @@
 
 #include "../graph_common.hpp"
 
-#include <sycl/detail/host_task_impl.hpp>
+#include <sycl/detail/core.hpp>
```

Comment:
We already include `<sycl/detail/core.hpp>` in [graph_common.hpp](https://github.com/intel/llvm/blob/1d9e1f5205cef6f271febaad210237a33a6fd0db/sycl/test-e2e/Graph/graph_common.hpp) don't think we need to add it again to all these graph tests.